primefaces/src/main/resources/META-INF/resources/primefaces/datepicker/0-datepicker.js

```diff
@@ -1474,7 +1483,7 @@
         },
 
         calculateWeekNumber: function(d) {
-            var dow = this.options.locale.firstDay !== undefined ? this.options.locale.firstDay : this.options.locale.firstDayOfWeek,
+            var dow = this.getFirstDayOfWeek(),
```

Comment:
I think this variable name is misleading. Isn't something like `firstDow` better?

Reply:
Fixed